### PR TITLE
Conditionals and Notebook UI fixes

### LINKED ISSUE #2853: [Conditionals] Condition Widget URL doesn't "blank out" properly
1. Create a new Condition Widget, don't define a URL in the Properties dialog. Save it.
1. Edit the widget and enter a URL. Save it, click the URL and observe that it links to the entered URL as expected.
1. Edit the widget and blank out the URL. Save it, then click the widget: observe it will navigate to My Items (or the root node). It shouldn't navigate anywhere.

### LINKED ISSUE #2867: Don't display View Switcher when editing an object
When editing a Condition Set, the view switcher control from Conditions View to Telemetry Table is available, but shouldn't be:
Screen Shot 2020-03-26 at 2 48 30 PM

This goes for any editable domain object.

### PR DESCRIPTION
**Pull request by charlesh88**
### Overview
Changes and fixes related to Conditionals and Notebook; fixes some issues in #2837 and #2796.

### Changes
- Significant fixes for Safari-compatible Flex layout in Condition Set view;
- Changed visual approach to current-value section in Condition Set view;
- Fix various layout issues in Firefox;
- Consolidate Conditionals styles into single scss file;
- Fix test datum elements layout, better wrapping;
- Fixes #2853 - better approach to presence/absence of URL property in Condition Widget;
- Fixes #2867 - hide the View Switcher when an object is being edited;
- Refined titling on View Switcher and Notebook menu button;
- Cleaned up styles in l-browse-bar and moved into ui/layout/layout.scss;
- Deleted styles/_layout.scss;
- Fixes regression that was allowing the main view Edit button to appear for mobile;

### Author Checklist
1. Changes address original issue? Y
1. Unit tests included and/or updated with changes? N/A
1. Command line build passes? Y
1. Changes have been smoke-tested? Y

### CODE REVIEW

**Comment by charlesh88** [comment]
> Looks like issues in mobile view when editing a condition set
> Unable to scroll the viewport at all.

You shouldn't be able to edit a Condition Set in mobile. Taking a look...

File: src/plugins/conditionWidget/components/ConditionWidget.vue [hunk 1]
```diff
@@ -21,13 +21,14 @@
 *****************************************************************************/
 
 <template>
-<a class="c-condition-widget"
-   :href="internalDomainObject.url"
+<component :is="cwHasUrl ? 'a' : 'span'"
```
Comment (charlesh88) [inline src/plugins/conditionWidget/components/ConditionWidget.vue:24]: Change this from a method to computed.

File: src/plugins/notebook/components/notebook-menu-switcher.vue [hunk 2]
```diff
@@ -1,8 +1,8 @@
 <template>
-<div class="l-browse-bar__view-switcher c-ctrl-wrapper c-ctrl-wrapper--menus-left">
+<div class="c-menu-button c-ctrl-wrapper c-ctrl-wrapper--menus-left">
     <button
         class="c-button--menu icon-notebook"
-        title="Switch view type"
+        title="Take a Notebook Snapshot"
```
Comment (nikhilmandlik) [inline src/plugins/notebook/components/notebook-menu-switcher.vue:5]: Thanks @charlesh88 for this change 👍

**Comment by charlesh88** [comment]
> > Looks like issues in mobile view when editing a condition set
> > Unable to scroll the viewport at all.
> 
> You shouldn't be able to edit a Condition Set in mobile. Looks like the Edit button now visible in mobile, I'll fix that.

Fixed.

File: src/plugins/conditionWidget/components/ConditionWidget.vue (hunk 1 above)
Comment (charlesh88) [reply src/plugins/conditionWidget/components/ConditionWidget.vue]: Fixed.

**Pull request merged**